deployment/console-backend-deploy.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: console-backend
+  labels:
+    app.kubernetes.io/component: console-backend
+    app.kubernetes.io/name: console-backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: console-backend
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: console-backend
+    spec:
+      serviceAccountName: console-backend
+      initContainers:
+      - name: wait-for-console-db
+        image: registry.redhat.io/rhel9/mariadb-105@sha256:050dd5a7a32395b73b8680570e967e55050b152727412fdd73a25d8816e62d53
+        command:
+        - /bin/sh
+        - -c
+        - |
+          until mysqladmin ping -hconsole-db --silent; do
+            echo 'Waiting for the console database to be ready...'
+            sleep 5
+          done
+      containers:
+      - name: console-backend
+        image: quay.io/securesign/rhtas-console@sha256:75966d60ed709af33efd48c53b96ea7b2fcd4608f90ccc56885bf224e34b55f5
```

Comment:
Could you load images from the [.env](https://github.com/securesign/rhtas-console-ui/blob/main/.env) file as we do on operator to avoid maintaining duplicity? https://github.com/securesign/secure-sign-operator/blob/main/config/default/images.env